meetings/2021-11-03.md

```diff
@@ -0,0 +1,204 @@
+
+* KK: I haven't contested it yet but have reservations. That is basically.. I'm still not sure what Tim is trying to achieve. There may be a misunderstanding of SPARQL. would like to clear that possibility out first.
+
+* RV: That may be a misunderstanding.. but we can still get a proper subset of SPARQL or any deviation.
+
+* KK: If you know what triple it is that you want to change. That is easy.. basically. It is also legitimate to change the triple and insert ??? If it is something else/more, then yes, we probably shouldn't be going with (that?) SPARQL now.
+
+* RV: Open question is: is this suitable for 1.0 or go for undisputed approach for 0.9.
+
+* KK: Perhaps. Would you implement it in CSS?
+
+* RV: The reason Tim wanted this.. he said you have 3 weeks to do it. Not my proudest work but it is simple. It is extensible. The support for N3 libs and support for CSS would be trivial. It is a low hanging fruit with a lot of option for extensibility. Lowest common demoniator so to speak.
+
+* KK: What do you all think?
+
+* SC: Go for it until someone
+
+* JB: ??? +1 n3 patch support
+
+* DZ: +1
+
+* KK: Doesn't it also mean we are deprecating the behaviour (in NSS)?
+
+* RV: I'd say there is disagreement around the behaviour. It is not stable yet. The stable version could be N3.
+
+* KK: At risk word would be use
+
+* DZ: What does solidOS use?
+
+* RV: SPARQL Patch. All of the code that's needed is already there.
+
+* KK: That's the client side? Right.. okay.
+
+* SC: General agreement to support N3 patch.
+
+* RV: Do I need to contribute to spec text?
+
+* KK: Yes. That'd be good.
+
+ACTION: RV to make initial draft. KK to assign relevant issues to RV. KK to record 'something' on that issue.
+
+
+### Write methods as requirements
+URL: https://github.com/solid/specification/pull/304
+
+* KK: Also for testing for compliance... if we have to test for eventuality that Solid server that does not support write operations then that makes the test suite more complex. We would want to support servers that are read-only but that is mostly just a Web server right.. At some point we want to introduce conformance classes. In addition, we want to have language that allows servers for read-only. Which is also in the spec.. Therefore to not tackle more than this then I'd like write methods for required.
+
+* RV: That clarifies a lot. I can imagine read-only servers. How do we know if a solid server announces whether htey are read or write?
+
+* KK: accept headers.
+
+* RV: or status codes.
+
+* SC: Test suite shouldn't be driving factor for the requirement
+
+* KK: I disagree with that idea because. The test suite is important for the health of the ecosystem. As a principle that is important but if there is no easy way to automate the testing then it is a toy.. you don't have a production system you have a toy.
```

Comment:
```suggestion
* KK: I disagree with that idea because the test suite is important for the health of the ecosystem. As a principle, that is important, but if there is no easy way to automate the testing then it is a toy. you don't have a production system, you have a toy.
```